BLSQ/iaso
# IA-2727 Feat: New field change_request_mode on Form

Feat: New field change_request_mode on Form

Modification to the "api/instances" endpoint: It the value of that field is CR_MODE_IF_REFERENCE_FORM it causes a change request to be created if the form is a reference form for the current org unit.

Related JIRA tickets : IA-2727 

## Self proofreading checklist

- [x ] Did I use eslint and black formatters
- [ x] Is my code clear enough and well documented
- [ x] My migrations file are included
- [x ] Are there enough tests
<br/>
<br/>
## Doc
- in the wiki

## How to test

Activate the flag on a form, ensure that it is a reference form, upload an instance and check that a change request is automatically created

## Print screen / video

Upload here print screens or videos showing the changes
